### ISSUE TITLE
VRTK_Body Physics Newbie question

### ISSUE THREAD

**Issue by Ugleez** [thestonefox/VRTK#872]
Hello, 

I can't find a solution how to avoid some collision, i tried with layer "Ignore raycast" didn't work in my case.
Then i tried with making tags, at least i tried to make a code to fix the problem, but it still doesnt work. I'm even not sure tags would work in that case. 

 private void StopCollision(GameObject inventory, bool state)
        {
            inventory = GameObject.FindWithTag("Inventory");
            if (inventory)
            {

                IgnoreCollisions(inventory.GetComponentsInChildren<Collider>(), true);
                IgnoreCollisions(inventory.GetComponents<Collider>(), true);

            }

Basically i have a bullet spawning in the area of the body collider and that makes the character jumping everywhere. Also i'm using another code for grabbing some object and i'd like to avoid collision with both. 

If anyone would help, i'd really appreciate. 
My apologise if my english is not that good.
Thank you,

**Comment by Ugleez** [comment]
Solved
